docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Controller/CgovMediaController.php

```diff
@@ -22,28 +22,24 @@ protected function getModuleName() {
   /**
    * Return an infographic's long description in an HTTP Response.
    *
-   * @param Drupal\media\Entity\Media $media
+   * @param \Drupal\media\Entity\Media $media
    *   Id of the infographic to render.
    */
   public function longDescription(Media $media) {
 
     if ($media == NULL) {
       throw new NotFoundHttpException();
     }
-
     // Only retrieve the field if it's available in the current language.
     $language = $this->languageManager()->getCurrentLanguage()->getId();
     if ($media->hasTranslation($language)) {
       $media = $media->getTranslation($language);
-
       // Only return a value if the field has a value. If it's missing or empty,
       // fall through to a NotFoundException.
-      if (count($media->field_accessible_version) > 0) {
-
-        $field = $media->field_accessible_version[0];
+      if ($media->hasField('field_accessible_version') && count($media->get('field_accessible_version')) > 0) {
+        $field = $media->get('field_accessible_version')[0];
```

Comment:
Can you change this to `$media->get('field_accessible_version')->first();`

Reply:
Resolved.